libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java

```diff
@@ -182,6 +187,8 @@ private void updateArrivalTime(RouteProgress routeProgress) {
       currentArrivalTime = TimeUtils.formatArrivalTime(routeProgress.durationRemaining());
       notificationRemoteViews.setTextViewText(R.id.notificationArrivalText,
         String.format(Locale.getDefault(), "%s ETA", currentArrivalTime));
+      bigNotificationRemoteViews.setTextViewText(R.id.notificationArrivalText,
+        String.format(Locale.getDefault(), "%s ETA", currentArrivalTime));
```

Comment:
We should extract `ETA` from here and internationalize it (i.e. add it to the translations).

Reply:
Why is not `ETA` appearing at all (not shown in the above screenshots)? 🤔